fence/README.md

```diff
@@ -0,0 +1,156 @@
+# Network Fencing
+
+## Terminology
+
+| Term     | Definition                                                                            |
+| -------- | ------------------------------------------------------------------------------------- |
+| CO       | Container Orchestration system that communicates with plugins using CSI service RPCs. |
+| SP       | Storage Provider, the vendor of a CSI plugin implementation.                          |
+| DR       | Disaster Recovery.                                                                    |
+| RPC      | [Remote Procedure Call](https://en.wikipedia.org/wiki/Remote_procedure_call).         |
+
+## Objective
+
+Define a standard that will enable storage vendors (SP) to
+perform node level fencing using corresponding CIDR blocks.
+
+### Goals in MVP
+
+The new extension will define a procedure that allows SP to block
+access to any client using corresponding CIDR blocks, and unblock
+as per the requirement.
+
+If multiple COs are accessing a shared resource provided by the SP, and a node
```

Comment:
is it multiple COs or Nodes ?

Reply:
Can be multiple COs too incase of a metro DR setup.